dev/create-release/create-release.sh

```diff
@@ -122,8 +122,14 @@ if [[ ! "$@" =~ --package-only ]]; then
   for file in $(find . -type f)
   do
     echo $GPG_PASSPHRASE | gpg --passphrase-fd 0 --output $file.asc --detach-sig --armour $file;
-    gpg --print-md MD5 $file > $file.md5;
-    gpg --print-md SHA1 $file > $file.sha1
+    if [ `command -v md5` ]; then
```

Comment:
Minor style comment: Could we use `$(...)` instead of backticks? I believe use of backticks is discouraged in bash.